.github/ISSUE_TEMPLATE/onboarding.md

```diff
@@ -8,32 +8,101 @@ assignees: ''
+- [ ] Assigned this ticket to the onboarded employee
 
-Install the following prerequisites:
+## Onboarded employee actions
 
-- [ ] [git-secrets](..#211-git-secrets)
+- [ ] Your UCSC account email: [ENTER-EMAIL-HERE]
+- [ ] Your GitHub account username: [ENTER-USERNAME-HERE]
+- [ ] Read [Azul README.md](..)
+- [ ] Read [Contributing Guidelines](../blob/develop/CONTRIBUTING.rst)
+- [ ] Read [AWS Credential Configuration](https://giwiki.gi.ucsc.edu/index.php/Overview_of_Getting_and_Using_an_AWS_IAM_Account)
+- [ ] Completed the NIH [Information Security and Management Refresher](https://irtsectraining.nih.gov/publicUser.aspx) training courses
+- [ ] Completed the NIH [Information Security Awareness for New Hires](https://irtsectraining.nih.gov/publicUser.aspx) training courses
+- [ ] Completed the NIH [Information Management for New Hires](https://irtsectraining.nih.gov/publicUser.aspx) training courses
+- [ ] Completed the [dbGaP Code of Conduct](https://www.surveymonkey.com/r/HKXNYD7) quiz
+- [ ] Completed the [NIH Security Best Practices for (CAD) Subject to NIH (GDS) Policy](https://www.surveymonkey.com/r/FG3C63T) quiz
+- [ ] Digitally signed the [NIH GDS Policy](https://giwiki.gi.ucsc.edu/images/1/1b/NIH_GDS_Policy.pdf) document and gave it to your project manager
+- [ ] Installed [git-secrets](..#211-git-secrets)
+- [ ] Completed steps for [setting up a VPN client](..#911-setting-up-a-vpn-client) for `dev.gitlab` and `anvildev.gitlab`
+- [ ] Sent a request email to the organization administrator (cc: hiring supervisor) asking them to:
+    - Create for you an AWS IAM user account
+    - Assign the `developer` role in the dev account(s) to the AWS IAM user account
+    - Assign the `developer` role to your GitHub user account
+    - Add your UCSC account to the `firecloud-cgl` group in Terra
+- [ ] Assigned this ticket to the hiring supervisor
 
-Request access to cloud infrastructure and tools:
+## Hiring supervisor actions
 
-- [ ] @theathorn: [UCSC-CGL Slack](ucsc-cgl.slack.com) through your Project Manager
- 
-- [ ] @hannes-ucsc: Push access to the Azul Repository on [#team-boardwalk](https://ucsc-cgl.slack.com/archives/C705Y6G9Z)
+- [ ] Approved (by replying to) the request email sent by the onboarded employee to the organization administrator
+- [ ] Assigned this ticket to the organization administrator
 
-- [ ] @hannes-ucsc: Access to development CI/CD systems on [#team-boardwalk](https://ucsc-cgl.slack.com/archives/C705Y6G9Z)
+## Organization administrator actions
 
-- [ ] @theathorn, Erich Weiler [AWS Credentials](https://gi-gateway.signin.aws.amazon.com/console)
+- [ ] Verified the authenticity of the approval
+- [ ] Verified the GitHub user account is associated with the onboarded employee's UCSC account email
+- [ ] Created an IAM user account for the onboarded employee
+- [ ] Added the `developer` role to the onboarded employee's IAM user account
```

Comment:
Why do we need an email if we list the specific items here?

Reply:
The onboarded employee's email address is used elsewhere in the checklist, example:
- [ ] Verified the GitHub user account is associated with the onboarded employee's UCSC account email